cmd/keeper/cmd/keeper.go

```diff
@@ -312,6 +312,7 @@ func (p *PostgresKeeper) createPGParameters(db *cluster.DB) common.Parameters {
 		for _, synchronousStandby := range db.Spec.SynchronousStandbys {
 			synchronousStandbys = append(synchronousStandbys, common.StolonName(synchronousStandby))
 		}
+		numberSyncStandbys := len(synchronousStandbys)
```

Comment:
If not do it then then the following code will result in forming of incorrect "synchronous_standby_names", it will will be like "2 (standby1, standby2)": 
		for _, synchronousStandby := range db.Spec.ExternalSynchronousStandbys {
			synchronousStandbys = append(synchronousStandbys, synchronousStandby)
		}
By saving the size of synchronousStandbys we are avoiding the for compilation of the incorrect value

Reply:
`2 (standby1, standby2)` is correct. As the comment below in the code says all the provided synchronous standbys must be synchronous. This is what I explained previously here: https://github.com/sorintlab/stolon/pull/410#issuecomment-360140604

Reply:
@sgotti I'm just wanted to notify you that we are currently considering your point of view. Will return to you in a couple of days....